### ISSUE TITLE
Uploads: Upload a file: submit button needs more font contrast

### ISSUE THREAD

**Issue by NiquiCara** [rubyforgood/abalone#558]
The green submit button doesn't have enough font contrast. Making the green a little darker should work.

![file-upload-font-contrast](https://user-images.githubusercontent.com/45127691/102563851-3999db80-40a8-11eb-9edd-82d4b4b02a50.PNG)

**Comment by metamoni** [comment]
Hi,

I'd like to take this one
